detox/e2e/support/test_config.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+module.exports = {
+    serverUrl: process.env.SITE_URL || (process.env.IOS ? 'http://localhost:8065' : 'http://10.0.2.2:8065'),
```

Comment:
I'm guessing default is going to be android by the look of this. yes ?

Reply:
Yes, just random thought and no other reason  :)